contracts/story-nft/BaseOrgStoryNFT.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+import { IERC165 } from "@openzeppelin/contracts/interfaces/IERC165.sol";
+
+import { IOrgStoryNFT } from "../interfaces/story-nft/IOrgStoryNFT.sol";
+import { BaseStoryNFT } from "./BaseStoryNFT.sol";
+
+/// @title Base Story NFT with OrgNFT integration
+/// @notice Base Story NFT which integrates with the OrgNFT and StoryNFTFactory.
+abstract contract BaseOrgStoryNFT is IOrgStoryNFT, BaseStoryNFT {
+    /// @notice Organization NFT address (see {OrgNFT}).
+    address public immutable ORG_NFT;
```

Comment:
add this comments:
`/// @custom:oz-upgrades-unsafe-allow state-variable-immutable`

Reply:
[d737282](https://github.com/storyprotocol/protocol-periphery-v1/pull/109/commits/d737282ad16bd860d8253fd0475dbd31fa68a1d6)